irods_browser_app/views.py

```diff
@@ -123,24 +123,18 @@ def upload_add(request):
         return JsonResponse({'error': "Invalid file type: {ext}".format(ext=ext)},
                             status=status.HTTP_400_BAD_REQUEST)
     else:
-        homepath = irods_fnames_list[0]
-        # TODO: this should happen whether resource is federated or not
-        irods_federated = utils.is_federated(homepath)
```

Comment:
just making a note for memory that removing this is_federated call should speed up the latency we have been seeing with users.hysroshare.org -> cloud